script/Ciel/Day 13/13_01_02_CIEL13_7C.txt

```diff
@@ -828,42 +962,51 @@ Here, I'll give you one extra dose, on the house!
 [sha:836753594415c7d08e1f1e2e560a354f80ea720f]{
 -- Page 337, Offset 36261. Glued. Mods: @x.
 -- あ、でもこれはシャレだから、追加で射つのは止めときなさいねぇ？　今ですら破裂寸前なんだもの。もう一つ上を目指すなんて、ほんと、一瞬で融けちゃうから！”
- Ah, but this one's quite the hit, so I'd have to caution against taking it, okay? You already look like you're about to explode, so if you try to go up another stage, you're seriously going to melt away in an instant!"
+// One last mint wafer
+ Ah, this one packs a real punch though,
+ so I wouldn't recommend shooting up any more than you already have, okay?
+ As things stand you're %{i}thiiis%{/i} close to your breaking point.
+ If you start gunning for the next Rank up, it really will melt your face off!"
 }
 [sha:4f232d570e3254cf0374ac7e423572cd0a0b2327]{
 -- Page 338, Offset 36262.
 -- 「あ―――ああぁぁあああ！」
-"Ah―――AAaaAAAAH!"
+// Noel needs her juice
+"%{i}Ah%{/i}――%{i}AAaaAAAAH%{/i}!"
 }
 [sha:9f19d512b50ea49850556603b0c85654c26076b9]{
 -- Page 339, Offset 36263.
 -- 少女の体が変貌していく。
-The girl's body transforms.
+The girl's body begins to transform.
 }
 [sha:1699c0fe117a0fb8237acbd26e73093bf2231575]{
 -- Page 339, Offset 36264.
 -- 身体に新星が生まれ、爆発し、銀河になった。
-A nova is conceived within her body, erupting and then transforming into a galaxy.
+A star is born inside her,
+ goes supernova,
+ and mushrooms out to form whole new galaxy.
 }
 [sha:4d116961db8c71a8e0b5de1105a1d451ff1961c3]{
 -- Page 339, Offset 36265.
 -- そう錯覚するほどの内部変革。
-At least, that's what her internal transformation seemed like.
+Or at least, you'd be forgiven for thinking it had if you were watching her transformation.
 }
 [sha:2a71413bebbbcde572688cd8f0c2094e991e0e27]{
 -- Page 339, Offset 36266.
 -- あふれ出す呪いを抑えきれず、まず真っ先に少女自身が崩壊した。
-Unable to contain the overflowing curse, her body begins to collapse before anything else.
+Her body immediately begins to crumble,
+ unable to contain the overflowing curse.
 }
 [sha:10d2e9b769e58a4d1dc359d98c8cc2020e09d725]{
 -- Page 340, Offset 36267.
 -- それでもソレには歓びがあった。
-But even so, there was still joy to be had.
+But that doesn't mean it was all for nothing.
 }
 [sha:d72d587949071617b34389c791d6390912dd9bb7]{
 -- Page 340, Offset 36268.
 -- 戦える。これで、あの女と戦える―――！
-She can fight. With this, she can finally oppose that woman―!
+She can fight.
+ With this, she finally stands a chance against that woman―――!
 }
 [sha:f679ab68c899d645398e6e2f34f7c065a679a5b8]{
 -- Page 341, Offset 36269.
```

Comment:
The ellipses aren't gone in 13/14 because I didn't want to deal with the merge conflicts, but there's a currently commented out lint rule that I'm going to enable for each day as we finish it, after which I'll circle back and get any it flags.